tests/App/config.yml

```diff
@@ -0,0 +1,51 @@
+framework:
+    secret: secret
+
+twig:
```

Comment:
I guess twig is not needed

Reply:
Removed the config, still had to load the bundle though... dependency to it somewhere...

Reply:
[This error](https://travis-ci.org/sonata-project/SonataMediaBundle/jobs/630007957#L804) is because of the path defined below does not exist, I would remove the config.